TypeCobol/Compiler/Sql/VisitorPattern.cs

```diff
@@ -29,6 +29,7 @@ public interface ISqlVisitor
         bool Visit([NotNull] UnpackedRowSelection unpackedRowSelection);
         bool Visit([NotNull] TableOrViewAllColumnsSelection tableOrViewAllColumnsSelection);
         bool Visit([NotNull] SqlExpression sqlExpression);
+        bool Visit([NotNull] SqlObject sqlObject);
```

Comment:
This is unnecessary. For `SqlObject`, the visit is handled through the `BeginSqlObject`, `EndSqlObject` and `VisitSqlObject` methods.
For classes that derive from `SqlObject`, a dedicated visit method is added.